common/src/main/java/com/revenuecat/purchases/common/offerings/OfferingsCache.kt

```diff
@@ -0,0 +1,61 @@
+package com.revenuecat.purchases.common.offerings
+
+import com.revenuecat.purchases.Offerings
+import com.revenuecat.purchases.common.DateProvider
+import com.revenuecat.purchases.common.DefaultDateProvider
+import com.revenuecat.purchases.common.caching.DeviceCache
+import com.revenuecat.purchases.common.caching.InMemoryCachedObject
+import com.revenuecat.purchases.common.caching.isStale
+import org.json.JSONObject
+
+class OfferingsCache(
```

Comment:
I really like this. I'd like to split `DeviceCache` in iOS as well so it's simply an implementation detail that we use in other cache classes like this one 👍🏻